source/Calamari/TerraformCliExecutor.cs

```diff
@@ -153,18 +155,30 @@ void InitializePlugins()
         {
             var initParams = variables.Get(TerraformSpecialVariables.Action.Terraform.AdditionalInitParams);
             var allowPluginDownloads = variables.GetFlag(TerraformSpecialVariables.Action.Terraform.AllowPluginDownloads, true);
+            string initCommand = $"init -no-color {initParams}";
+
+            if (version.CompareTo(new Version("0.15.0")) < 0)
```

Comment:
The scourge of CompareTo - does it mean greater than, or less than? Could we put an extension method on `Version` so this was simply `version.IsLessThan("0.15.0")` and shift the rest of the logic / new version instantiation into that? :)

Reply:
Then we could simplify the whole thing to
```
string initCommand = version.IsLessThan("0.15.0") ? $"init -no-color {initParams}" : $"init -no-color -get-plugins={allowPluginDownloads.ToString().ToLower()}
```

Reply:
I can make it 
```
string initCommand = $"init -no-color{(version.IsLessThan("0.15.0") ? $" -get-plugins={allowPluginDownloads.ToString().ToLower()}" : "")} {initParams}";
```
But I find it to be less readable, makes it too easy to miss what's going on here.  In general I prefer not to use ternary operator when the resulting line is more than a few dozen characters.

Reply:
Sure - happy to skip the ternary, I'm more interested in the extension method :)